webpack/scenes/ContentViews/helpers.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+import {
+  Tooltip,
+  TooltipPosition,
+  Flex,
+  FlexItem,
+} from '@patternfly/react-core';
+import ContentViewIcon from './components/ContentViewIcon';
+
+const HelpToolTip = tooltip => (
+  <span className="foreman-spaced-icon">
+    <Tooltip
+      position={TooltipPosition.top}
+      content={tooltip}
+    >
+      <OutlinedQuestionCircleIcon />
+    </Tooltip>
+  </span>
+);
+
+export const dependenciesHelpText = __('This will solve RPM and module stream dependencies on every publish of this content view. ' +
+  'Dependency solving significantly increases publish time (publishes can take over three times as long) ' +
+  'and filters will be ignored when adding packages to solve dependencies. ' +
+  'Also, certain scenarios involving errata may still cause dependency errors.');
+
+export const autoPublishHelpText = __('Automatically publish a new version of the composite content view whenever one of its content views is published. ' +
+  'Autopublish will only happen for component views that use the \'Always use latest version\' option.');
+
+export const importOnlyHelpText = __('Designate whether this content view is for importing from an upstream server. ' +
+  'Import-only content views can not be published directly.');
+
+
+export const LabelComposite = () => (
```

Comment:
Are these `Label*` components really helpers? I'm only seeing them used in one place and I don't think of a helpers file having components, I think they could be in the file they are being used in or a separate file.

Reply:
Re-using LabelAutoPublish in the info page right now. I think you wanted to move it out of the form page in a separate file and I had it in a create_form_component.js file but then moved it to helper file to keep it together..